types/node/events.d.ts

```diff
@@ -260,6 +268,26 @@ declare module 'events' {
          * @since v15.2.0, v14.17.0
          */
         static getEventListeners(emitter: DOMEventTarget | NodeJS.EventEmitter, name: string | symbol): Function[];
+        /**
+         * By default `EventEmitter`s will print a warning if more than `10` listeners are
+         * added for a particular event. This is a useful default that helps finding
+         * memory leaks. The `EventEmitter.setMaxListeners()` method allows the default limit to be
+         * modified or modify the limit specified in every `EventTarget` passed as arguments.
+         * The value can be set to`Infinity` (or `0`) to indicate an unlimited number of listeners.
+         *
+         * ```js
+         * EventEmitter.setMaxListeners(20);
+         * // Equivalent to
+         * EventEmitter.defaultMaxListeners = 20;
+         *
+         * const eventTarget = new EventTarget();
+         * // Only way to increase limit for `EventTarget` instances
+         * // as these doesn't expose its own `setMaxListeners` method
+         * EventEmitter.setMaxListeners(20, eventTarget);
+         * ```
+         * @since v15.3.0, v14.17.0
+         */
+        static setMaxListeners(n?: number, ...eventTargets: DOMEventTarget[]): void;
```

Comment:
Shouldn't this allow both types, as in documentation and ,say, in `getEventListeners`?:
```ts
static getEventListeners(emitter: DOMEventTarget | NodeJS.EventEmitter
```
we support v14, v16 and current, so this should added all versions (as per support coment)

Reply:
Yes it should support EventEmitters too.

Reply:
Also please ensure to add the note what happens when `eventTargets` is empty.